lib/basket_item.rb

```diff
@@ -11,6 +11,8 @@ def increase_quantity
   end
 
   def decrease_quantity
+    raise StandardError unless @quantity > 0
```

Comment:
will be better to add specific error to describe it, like:

QuantityLevelError = Class.new(StandardError)